client/a8c-for-agencies/sections/marketplace/pressable-overview/plan-selection/details.tsx

```diff
@@ -0,0 +1,153 @@
+						{ translate( '%(planName)s plan', {
+							args: {
+								planName: selectedPlan ? getPressableShortName( selectedPlan.name ) : customString,
+							},
+							comment: '%(planName)s is the name of the selected plan.',
+						} ) }
+					</h3>
+
+					{ selectedPlan && (
+						<div className="pressable-overview-plan-selection__details-card-header-price">
+							<strong className="pressable-overview-plan-selection__details-card-header-price-value">
+								{ formatCurrency( discountedCost, selectedPlan.currency ) }
+							</strong>
+							<span className="pressable-overview-plan-selection__details-card-header-price-interval">
+								{ selectedPlan.price_interval === 'day' && translate( 'per plan per day' ) }
+								{ selectedPlan.price_interval === 'month' && translate( 'per plan per month' ) }
+							</span>
+						</div>
+					) }
+				</div>
+
+				<IncludedList
+					items={ [
+						translate( '{{b}}%(count)s{{/b}} WordPress installs', {
+							args: {
+								count: info?.install ?? customString,
+							},
+							components: { b: <b /> },
+							comment: '%(count)s is the number of WordPress installs.',
+						} ),
+						translate( '{{b}}%(count)s{{/b}} visits per month', {
+							args: {
+								count: info ? formatNumber( info.visits ) : customString,
+							},
+							components: { b: <b /> },
+							comment: '%(count)s is the number of visits per month.',
+						} ),
+						translate( '{{b}}%(size)s{{/b}} storage per month', {
+							args: {
+								size: info ? `${ info.storage }GB` : customString,
+							},
+							components: { b: <b /> },
+							comment: '%(size)s is the amount of storage in gigabytes.',
+						} ),
+					] }
+				/>
+
+				{ selectedPlan && (
+					<Button
+						className="pressable-overview-plan-selection__details-card-cta-button"
+						onClick={ onSelectPlan }
+						variant="primary"
+					>
+						{ translate( 'Select %(planName)s plan', {
+							args: {
+								planName: selectedPlan ? getPressableShortName( selectedPlan.name ) : customString,
+							},
+							comment: '%(planName)s is the name of the selected plan.',
+						} ) }
+					</Button>
+				) }
+
+				{ ! selectedPlan && (
+					<Button
+						className="pressable-overview-plan-selection__details-card-cta-button"
+						onClick={ onChatWithUs }
+						href={ PRESSABLE_CONTACT_LINK }
+						target="_blank"
+						variant="primary"
+					>
+						{ translate( 'Chat with us' ) } <Icon icon={ external } size={ 16 } />
+					</Button>
+				) }
+			</div>
+
+			<div className="pressable-overview-plan-selection__details-card is-aside">
+				<h3 className="pressable-overview-plan-selection__details-card-header-title">
+					{ translate( 'All plans include:' ) }{ ' ' }
+				</h3>
+
+				<IncludedList
+					items={ [
+						translate( '24/7 WordPress hosting support' ),
+						translate( 'WP Cloud platform' ),
+						translate( 'Jetpack Security (optional)' ),
+						translate( 'Free sites migration' ),
```

Comment:
I think there might be a typo here. Should this be "Free site migration" (instead of "Free **sites** migration" (plural sites))

Reply:
Good catch. Thanks for pointing this out. will fix this on a separate PR 👍

Reply:
https://github.com/Automattic/wp-calypso/pull/89068